### DISCUSSION TITLE
How to expand the vertical height of the chat window?

### DISCUSSION THREAD

**Discussion by shijunbao** [oobabooga/text-generation-webui#3560] [General]
How to expand the vertical height of the chat window?

I want to obtain twice the height of the chat window, which is more convenient for viewing chat records.

![image](https://github.com/oobabooga/text-generation-webui/assets/67893042/50603b09-0aee-49b9-ba3d-2bebeaba0cab)

**Comment by missionfloyd** [comment]
https://github.com/oobabooga/text-generation-webui/blob/3ae2cee446b075a20937a60d4e49d42bebf4a744/css/main.css#L257
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Values_and_Units#dimensions
